.github/workflows/autoupdate-preview.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
         run: make fetch_openapi
         env:
           API_BASE_URL: ${{ secrets.MDB_CURRENT_API_PREVIEW_URL }}
```

Comment:
This can also be removed since we no longer using mongodb cloud as source.

Reply:
```suggestion
          API_BASE_URL:  https://raw.githubusercontent.com/mongodb/openapi/refs/heads/dev/openapi/v2
```

Reply:
We can hardcode it as you only need 2 values and each of those values are in separate github action yaml.
The reason this was secret was that we did not wanted to expose our non public url. The urls you are using are public.